src/components/PaginationItem/PaginationItem.tsx

```diff
@@ -0,0 +1,24 @@
+interface Props {
+  page: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const PaginationItem: React.FC<Props> = ({
+  page,
+  currentPage,
+  setCurrentPage,
+}) => {
+  return (
+    <li className={`page-item ${currentPage === page ? 'active' : ''}`}>
+      <a
+        data-cy="pageLink"
+        className="page-link"
+        onClick={() => setCurrentPage(page)}
+        href={`#page-${page}`}
```

Comment:
It's important to provide a `role` attribute to interactive elements for accessibility purposes. Since this is acting as a button, you can add `role='button'` to the anchor tag.